projects/core/src/state/services/state-persistance.service.spec.ts

```diff
@@ -34,7 +34,7 @@ const winRef = {
   },
 } as WindowRef;
 
-describe('StatePersistanceService', () => {
```

Comment:
Thanks for contribution. Can we also rename the files?

Reply:
Renamed file from state-persistance.service.spec.ts to state-persistence.service.spec.ts